### PR TITLE
Added Python for AoC

### PR DESCRIPTION
**Pull request by cobanov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,7 +229,6 @@ Read [CONTRIBUTING.md](/CONTRIBUTING.md) to learn how to add your own repos.
 
 *Solutions to AoC in Erlang.*
 
-* [hypothermic/advent-of-code-2021](https://github.com/hypothermic/advent-of-code-2021) ![Last Commit on GitHub](https://img.shields.io/badge/last%20commit-2021--12--01-red)
```
Comment (Bogdanp) [inline README.md:232]: Why was this removed?

**Review by Bogdanp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cobanov) [reply README.md:232]: I didnt see, sorry idk how it is happened, let me fix
